packages/cli/package.json

```diff
@@ -53,6 +53,8 @@
     "inquirer": "6.5.0",
     "klaw": "3.0.0",
     "lodash": "4.17.15",
+    "marked": "0.8.0",
```

Comment:
When running `yarn`, I get `warning "workspace-aggregator-... > zapier-platform-cli > marked-terminal@3.3.0" has incorrect peer dependency "marked@^0.4.0 || ^0.5.0 || ^0.6.0 || ^0.7.0".` I'd probably use `marked@0.7.0` unless there's a big reason to use 0.8.0.